### PR TITLE
💄[Component] Update SearchResults

### PR DESCRIPTION
**Pull request by AuroraHuang22**
1. Add Video, Text, CreativeContent icons
2. Update SearchResult.vue
3. Use conditional to insert icon component

### CODE REVIEW

File: components/SearchResults.vue [hunk 1]
```diff
@@ -1,31 +1,79 @@
...
+                />
+                <IconText
+                  v-if="record.data.contentMetadata['@type'] === 'Text'"
+                />
+                <IconText
+                  v-if="record.data.contentMetadata['@type'] === 'Article'"
+                />
+                <IconVideo
+                  v-if="record.data.contentMetadata['@type'] === 'Video'"
+                />
```
Comment (nwingt) [inline components/SearchResults.vue:56]: Can refactor

```suggestion
                <ISCNTypeIcon :type="record.data.contentMetadata['@type']" />
```

```html
<!-- ISCNTypeIcon.vue -->
<IconText v-if="recordType === 'text" />
<IconImage v-if="recordType === 'image" />
...
```
```javascript
get recordType() {
  switch (type) {
    case 'Image':
    case 'Photo':
      return 'image'
   ...
  }
}
```

File: pages/works/index.vue [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-    <div v-if="!records">
+  <div class="mx-auto pt-[16px]">
+    <template v-if="!records">
       {{ $t('general.loading') }}
-    </div>
-    <search-results v-else :records="records"/>
+    </template>
+    <template v-else>
+      <search-results :records="records" />
+    </template>
```
Comment (nwingt) [inline pages/works/index.vue:8]: Can remove extra template?

File: pages/view/_iscnId.vue [hunk 3]
```diff
@@ -16,16 +16,32 @@
...
-        :time-stamp="record.recordTimestamp"
-      >
+      <InfoCard :label-text="type" :time-stamp="record.recordTimestamp">
         <template #icon>
-          <IconImage />
+          <IconImage v-if="type === 'Photo'" />
+          <IconImage v-if="type === 'Image'" />
+          <IconText v-if="type === 'Text'" />
+          <IconText v-if="type === 'Article'" />
+          <IconVideo v-if="type === 'Video'" />
```
Comment (nwingt) [inline pages/view/_iscnId.vue:44]: Same as above

**Review by nwingt — CHANGES_REQUESTED** [review]

File: components/SearchResults.vue (hunk 1 above)
Comment (AuroraHuang22) [reply components/SearchResults.vue:56]: 🤩🤩🤩

File: components/ISCNTypeIcon.vue [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <IconImage v-if="recordType === 'Image'" />
+    <IconText v-if="recordType === 'Text'" />
+    <IconVideo v-if="recordType === 'Video'" />
+    <IconCreativeContent v-if="recordType === 'CreativeContent'" />
+  </div>
```
Comment (nwingt) [inline components/ISCNTypeIcon.vue:7]:
```suggestion
  <IconImage v-if="recordType === 'Image'" />
  <IconText v-if="recordType === 'Text'" />
  <IconVideo v-if="recordType === 'Video'" />
  <IconCreativeContent v-else />
]```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: pages/view/_iscnId.vue [hunk 5]
```diff
@@ -16,16 +16,13 @@
       'mb-[30px]',
     ]"
   >
-    <div class="mr-[32px]">
-      <MetadataCard :img-src="imgSrc" />
+    <div  class="mr-[32px]">
```
Comment (nwingt) [inline pages/view/_iscnId.vue:19]:
```suggestion
    <div class="mr-[32px]">
```

**Review by nwingt — CHANGES_REQUESTED** [review]

**Pull request merged**